src/components/lowering_replay.js

```diff
@@ -69,8 +66,12 @@ class LoweringReplay extends Component {
   }
 
   componentDidMount() {
+    const param = this.props.match.params.id;
+
     if(this.props.event.events.length === 0) {
-      this.props.initLoweringReplay(this.props.match.params.id, this.props.event.hideASNAP);
+      // initLoweringReplay now handles both database IDs and semantic lowering_ids
+      // It also fetches the cruise data automatically
```

Comment:
I am reluctant to have the codebase full of "x *now* does such and such": it is implicit that comments describe what the code does now, unless it is specifically a comment about a change in behavior for some reason like backwards compatibility.